server/routes/api/v1/getUsers.js

```diff
@@ -0,0 +1,33 @@
+const admin = require('firebase-admin');

```

Comment:
Is this file "getUsers.js" actually getting multiple users? It seems like it's just requesting a singular user from the database.

It would be good to update the name of the file to "getUser.js" if so.

Reply:
I can definitely rename it if it's unclear. 

The intuition was: each method on a route would have a filename of `<method><route>.js` for consistency, but I can see how this isn't always a good idea. 

Ex. You have `GET /api/v1/users` return a list of users, whereas `GET /api/v1/users/:uid` returns only one user. You could put both these routes in the same `getUsers.js`, or we could separate them into `getUsers.js` and `getUser.js`. The latter filename does not represent the exact route that's being requested, but it probably doesn't matter that much, and I'm sure there are more cases where this naming scheme would fall apart.

I'm going to rename it to `getUser.js`, like you reccomended, because as our API stands right now, that makes the most sense.